# Service to copy a log file into a folder.
string filename      # Name of the logFile to copy (realtive to the logFolder).
string folder        # Folder where to copy (relative to the logFolder)
---
int64 ret            # Set to 1 (success) or 0 (error)
string msg           # Error description
